src/cr/cube/dimension.py

```diff
@@ -954,6 +959,41 @@ def _order_dict(self):
         return self._dimension_transforms_dict.get("order") or {}
 
 
+class _PairwiseSignificanceSpec(object):
+    """Value object providing access to details of pairwise significance information."""
+
+    def __init__(self, dimension, dimension_transforms_dict):
+        self._dimension = dimension
+        self._dimension_transforms_dict = dimension_transforms_dict
+
+    @lazyproperty
+    def column_idx(self):
```

Comment:
I'd be thinking `.reference_column_idx`. By itself, `.column_idx` could mean a lot of things.